GOVERNANCE.md

```diff
@@ -1,137 +1,121 @@
+this project, including:
 
 * Technical direction
-* Project governance and process (including this policy)
+* Project governance
 * Contribution policy
 * GitHub repository hosting
-* Conduct guidelines
 * Maintaining the list of additional Collaborators
 
-* [Current list of TSC members](./README.md#current-project-team-members)
+* [Current list of TC members](./README.md#current-project-team-members)
 
-The operations of the TSC are governed by the [TSC Charter][] as approved by
-the Node.js Foundation Board of Directors.
+## Community Committee
 
-### TSC Meetings
+The Community Committee (CC) is responsible for handling issues regarding the
+broad community of the Ayo.js project. This includes, but is not limited to:
 
-The TSC meets regularly in a voice conference call. The meeting is run by a
-designated meeting chair approved by the TSC. Each meeting is streamed on
-YouTube.
+* Evangelism (this includes outward appearance, social media, etc)
+* Conferences
+* Meetups
+* Conduct
+* Moderation
 
-Items are added to the TSC agenda which are considered contentious or
-are modifications of governance, contribution policy, TSC membership,
-or release process.
+* [Current list of CC members](./README.md#current-project-team-members)
 
-The intention of the agenda is not to approve or review all patches.
-That should happen continuously on GitHub and be handled by the larger
-group of Collaborators.
+### Correspondence between TC and CC
 
-Any community member or contributor can ask that something be reviewed
-by the TSC by logging a GitHub issue. Any Collaborator, TSC member, or the
-meeting chair can bring the issue to the TSC's attention by applying the
-`tsc-review` label. If consensus-seeking among TSC members fails for a
-particular issue, it may be added to the TSC meeting agenda by adding the
-`tsc-agenda` label.
+The TC and the CC are normally completely independent from each other, however
+in the event where an issue arises that crosses both of their areas of
+expertise, a joint action is required. In these events, none of the two groups
+may make a final decision until the other agrees.
 
-Prior to each TSC meeting, the meeting chair will share the agenda with
-members of the TSC. TSC members can also add items to the agenda at the
-beginning of each meeting. The meeting chair and the TSC cannot veto or remove
-items.
+## Elected Final Arbiters
 
-The TSC may invite additional persons to participate in a non-voting capacity.
+Elected Final Arbiters (EFA) are Members of the Ayo.js projects that serve to
+resolve issues where there is no consensus. They are elected on a 1-year term,
+during which they can choose to resign at any time. In the event that this
+happens, reelections immediately taken into consideration.
 
-The meeting chair is responsible for ensuring that minutes are taken and that a
-pull request with the minutes is submitted after the meeting.
+There should always be multiple EFAs, a recommended number is 4.
```

Comment:
How about 3 or 5 to always have an odd number?

Reply:
This raises interesting issue - in case of different opinions of EFA there is a vote?